include/fail_safes.hpp

```diff
@@ -0,0 +1,95 @@
+#include "config.hpp"
+#include "motors.hpp"
+
+namespace bzzz
+{
+    class FailSafes
+    {
+        private:
+        /**
+         * Radio timeout variables for connection loss check
+        */
+       unsigned long long m_lastRadioReceptionTime;
+       unsigned long m_radioConnectionTimeoutInMicroseconds;
```

Comment:
The type of `m_lastRadioReceptionTime` is `unsigned long long`, but `m_radioConnectionTimeoutInMicroseconds` is `unsigned long`. Is that correct?

Reply:
Yes, professor. `m_lastRadioReceptionTime` needs to store up time in microseconds, which we don't know how long. While `m_radioConnectionTimeoutInMicroseconds` is just a positive constant.